Source/NetworkActivityIndicatorManager.swift

```diff
@@ -124,31 +126,29 @@ public class NetworkActivityIndicatorManager {
         invalidateCompletionDelayTimer()
     }
 
-    // MARK: - Activity Count
+    // MARK: - Request Tracking
 
-    /// Increments the number of active network requests.
+    /// Adds the requestID as an active request driving the activity indicator.
     ///
-    /// If this number was zero before incrementing, the network activity indicator will start spinning after
-    /// the `startDelay`.
+    /// This method results in a no-op if the request is already being tracked.
     ///
-    /// Generally, this method should not need to be used directly.
-    public func incrementActivityCount() {
+    /// - Parameter requestID: The request identifier.
+    public func requestDidStart(requestID: String) {
```

Comment:
I'd love to get your thoughts on this API @jshier. I went through a lot of iterations on the name of this API, and landed on `requestDidStart` and `requestDidStop`.

Reply:
Seems like the only other option would be `didStartRequest(id: String)`. Either way, we may want to make the parameter name `id`, as `requestID` seems a bit redundant.

Reply:
Yep. We have a convention at Nike where we would use `requestDidStart(withID requestID: String)` where we drop the `request` portion of the external API and supplement with `with` to make it read better. Then we keep the `request` on the internal parameter. Is that a convention that would be a better alternative than just `id`? We try to avoid `id` so that we always know exactly what `id` actually represents. In this case, our intention is that the `id` represents the `requestID`, but to your point, that can certainly be inferred given the rest of the API signature.

Anyways, I'm happy with any of the following:

- `requestDidStart(requestID: String) // option 1`
- `requestDidStart(id: String) // option 2`
- `requestDidStart(withID requestID: String) // option 3`

Do you have a strong preference on any?

Reply:
I guess it depends on whether you consider the `with` implicit in the fact that it's a parameter. I think 2 or 1 would work, depending on whether we need the disambiguation for `id`.

Reply:
Refactored API to `requestDidStart(id: String)` in bf277d73.

Reply:
Decided to stick with `requestDidStart(requestID: String)` based on Slack conversation to avoid ambiguity and make it very clear that the `id` is meant to be the `requestID`.